source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/ManagementSummaryView.tsx

```diff
@@ -0,0 +1,36 @@
+import { EditButton } from '@/components/common/EditButton';
+import { Section } from '@/components/common/Section/Section';
+import { StyledEditWrapper } from '@/components/common/Section/SectionStyles';
+import { Claims } from '@/constants/index';
+import useKeycloakWrapper from '@/hooks/useKeycloakWrapper';
+import { Api_Property } from '@/models/api/Property';
+import { Api_PropertyLease } from '@/models/api/PropertyLease';
+
+export interface IManagementSummaryViewProps {
+  isLoading: boolean;
+  property: Api_Property;
+  propertyLeases: Api_PropertyLease[];
+  setEditMode: (isEditing: boolean) => void;
+}
+
+export const ManagementSummaryView: React.FunctionComponent<IManagementSummaryViewProps> = ({
+  isLoading,
+  property,
+  propertyLeases,
+  setEditMode,
+}) => {
+  const { hasClaim } = useKeycloakWrapper();
+  return (
+    <Section header="Summary">
+      <StyledEditWrapper className="mr-3 my-1">
+        {/** TODO: Use MANAGEMENT CLAIMS when available */}
```

Comment:
@asanchezr the management claims are available, can you add those please?

Reply:
They are available now - not when I sent the PR originally. Did you add them to the frontend enum as well or do I need to do that?